### ISSUE TITLE
Multiline todo gets printed twice

### ISSUE THREAD

**Issue by hanneskaeufler** [hanneskaeufler/danger-todoist#28]
The added code:

```java
	/**
	 * TODO: NotFoundException should not really know how to map itself to a
	 * response.
	 */
	@Override
	public Response toResponse(NotFoundException exception) {
		return Response.status(Response.Status.NOT_FOUND).build();
	}
```

The resulting comment by danger:

<img width="719" alt="bildschirmfoto 2016-12-25 um 15 54 51" src="https://cloud.githubusercontent.com/assets/68024/21471694/932de906-caba-11e6-8504-b486a5f6f3f9.png">

What I expected:
Just one line, repeating the todo in the code

What I got:
The same test repeated twice.

**Comment by bright-star** [comment]
Also note that hard-wrapped TODOs with single-line comment syntax are printed once, but only with the first line:

```
// TODO This is a TODO that was wrapped
// around to this line
```

- This is a TODO that was wrapped

**Comment by hanneskaeufler** [comment]
This is resolved with #29. @bright-star I will sleep over this and cut a new release which solves the issue you mentioned. thanks for reporting 💌
